global_helpers/panther_oss_helpers.py

```diff
@@ -241,6 +241,17 @@ def set_key_expiration(key: str, epoch_seconds: int) -> None:
         key: The name of the counter
         epoch_seconds: When you want the counter to expire (set to 0 to disable)
     """
+    if isinstance(epoch_seconds, str):
+        epoch_seconds = float(epoch_seconds)
```

Comment:
dynamo's TTL  expects this to be an int ... so any floats we detect should be converted to an int

Reply:
indeed. check out lines 246 and 247 👍

Reply:
yeah its a minor performance thing, but if we saw a string, we'd be doing two casts instead of 1. The float check is good, just don't think we need to cast to a float at this point.